cmd/k8s-operator/deploy/chart/templates/deployment.yaml

```diff
@@ -34,17 +34,11 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- if or .Values.oauth.clientSecret .Values.oauth.audience }}
       volumes:
```

Comment:
This will be an empty (and invalid) block if none of clientSecret, audience, or oauthSecretVolume are set